vignettes/cookbook_interpretation.Rmd

```diff
@@ -14,59 +14,164 @@ knitr::opts_chunk$set(
 )
 ```
 
-## Coverage Diagnostics
-
-### Match Success Rate
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
-
-## PACTA Target Market Share Metrics
-
-### Sector-level technology mix
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
-
-### Technology-level volume trajectory
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
-
-## PACTA SDA Metric
-
-### Sector-level emission intensity
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
+## Target Market Share
+
+[brief description of Target Market Share metric]
+
+For detailed information about the Target Market Share metric and how it is calculated, see the [Metrics appendix](cookbook_metrics.html#market-share-approach).
+
+### `target_market_share()` function
+
+The `target_market_share()` function calculates the Target Market Share metric and outputs a table including the calculated targets at the portfolio level or at the company level, depending on the options selected.
+
+#### Explanation of Output
+
+The primary columns/variables of interest in the output are the `production` and `metric` variables. What the `production` variable represents depends on the values in all of the columns that precede it, and in particular the `metric` column. If the value in the `metric` column on the same row is `"projected"`, then the value in the `production` variable will represent the production level of your portfolio based on the ABCD data for that sector/technology/year/scenario/etc. If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `metric` column on the same row is `"corporate_economy"`, then the value in the `production` variable will represent [the total production levels of the entire sector adjusted by the start year production level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels].
```

Comment:
Why is the last part of this sentence in square brackets?

Reply:
because I'm not sure if it's true/correct

Reply:
I updated this section